CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed
+
+- `version` and `purl` fields in `components` list are now optional in CycloneDX SBOMs.
```

Comment:
```suggestion
- `version` and `purl` fields in `components` list are now optional in CycloneDX SBOMs
```

I was going to also suggest changing it up a bit to be below 80 columns, but iirc the cli changelog is more of a ~100 column limit and generally 80 can be a bit tight. So I think it should be fine as-is without the full stop.